src/resources/reviews.jsx

```diff
@@ -0,0 +1,59 @@
+import { 
+    List, 
+    ReferenceField,
+    TextField, 
+    Datagrid,
+    Edit,
+    SimpleForm,
+    ReferenceInput,
+    TextInput,
+    DateTimeInput,
+    Create,
+    NumberField,
+    NumberInput
+} from "react-admin";
+
+const reviewFilters = [
+  <TextInput source="q" label="Search" alwaysOn />,
+];
+
+export const ReviewList = () => (
+  <List filters={reviewFilters}>
+    <Datagrid rowClick="edit">
+      <TextField source="id" />
```

Comment:
Think this should show the text

Reply:
which text?

Reply:
I mean the review list should have a "text" column. The text is the most important thing after the rating.

Reply:
i thought that if the text it to long it's maybe a problem.

Reply:
yeah but I'm sure we can let it overflow gracefully in some way? maybe that's already the default

Reply:
i added it what do you think?

Reply:
Nice, thanks! Still think this should be truncated. We can write a custom field for that. Something like
```
const TruncatedTextField = ({ record }) => {
  let str = record.text;
  return record ? (
    <span>{str.length > 50 ?  str.slice(0,50) + "..." : str}</span>
  ) : null;
};
```

Reply:
did that thanks for the hint